omod/src/main/java/org/openmrs/web/dwr/DWRFormService.java

```diff
@@ -109,6 +109,14 @@ public FormFieldListItem getFormField(Integer formFieldId) {
 		for (Field field : Context.getFormService().getFields(txt)) {
 			FieldListItem htmlSafeField = new FieldListItem(field, Context.getLocale());
 			htmlSafeField.setName(Encode.forHtml(htmlSafeField.getName()));
+			htmlSafeField.setFieldTypeName(Encode.forHtml(htmlSafeField.getFieldTypeName()));
+			
```

Comment:
@annaowens  you can remove this space

Reply:
I think the line break enhances readability a bit by separating the name encoding from the description encoding. It doesn't look like it violates any [OpenMRS coding conventions](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle), either.